lib/lfrfid/tools/bit_lib.h

```diff
@@ -88,6 +88,14 @@ uint16_t bit_lib_get_bits_16(const uint8_t* data, size_t position, uint8_t lengt
  */
 uint32_t bit_lib_get_bits_32(const uint8_t* data, size_t position, uint8_t length);
 
+/**
+ * @brief Compute parity bit for a given byte
+ * @param bits Bits to compute the parity for
+ * @param odd Even or odd parity
+ * @return The parity bit
+ */
+uint8_t bit_lib_parity(const uint8_t bits, bool odd);
```

Comment:
bit_lib_test_parity_32 do the same.
I think that you just need to rename bit_lib_test_parity_32 -> bit_lib_test_parity. That will be better.

Reply:
Using bit_lib_test_parity_32